jupyterhub/handlers/pages.py

```diff
@@ -455,7 +456,9 @@ class AdminHandler(BaseHandler):
     """Render the admin page."""
 
     @web.authenticated
-    @admin_only
+    @needs_scope('users')  # stacked decorators: all scopes must be present
+    @needs_scope('admin:users')
+    @needs_scope('admin:servers')
```

Comment:
what is the difference to `@admin_only`? Is the new scope more or less permissive?

Reply:
`@admin_only` was removed from jupyterhub and was replaced by the scope module.